addons/sms/wizard/sms_composer.py

```diff
@@ -289,11 +289,15 @@ def _prepare_recipient_values(self, records):
         recipients_info = records._sms_get_recipients_info(force_field=self.number_field_name)
         return recipients_info
 
+    def _get_body_additional_context(self):
+        return dict()
```

Comment:
Would move to a new section called "Render", before "Tools". And apply it when calling "_compute_body" too for comment mode.

Maybe just call it `_get_additional_render_context` too, I suppose you might generally like to use it for other things than just the body